### ISSUE TITLE
Statistics Display - DAU & MAU or Real Time active users displayed via graph or number.

### ISSUE THREAD

**Issue by absolutep** [Tribler/tribler#7215]
I just got to know about this and was wondering why Tribler team does not update this on their official website or show this on the Tribler itself.

Link = https://release.tribler.org/dashboard/

This will massively help in regaining trust of the users which was lost due to a few unfortunate mis-happenings.

It can be shown as a graph similar to how token balance was colourfully displayed in previous versions.

Or via real time active user number

Or in a as-is format. Whichever suits the developers.

I literally thought not more than 1000 people in totality, knew about Tribler (due to those mishaps being the first thing coming into discussion about Tribler) let alone use it daily.
![Untitled](https://user-images.githubusercontent.com/97702149/206673959-3eca767e-eafb-44b6-9a4a-ea844c27af64.png)
![2](https://user-images.githubusercontent.com/97702149/206673985-b7f7aa78-e2e1-483d-b153-85998b0a276c.png)

Even though, it may seem trivial this will very much certainly help Tribler's popularity.

BiglyBT uses this feature to its advantage as it helps their selfmade DHT system to grow and index or connect to more peers.
![1](https://user-images.githubusercontent.com/97702149/206673397-7adc80b6-8354-4e42-bf14-709887434683.png)

**Comment by hbiyik** [comment]
i think the real problem about the popularity is not exactly the trust, but the network is very much centralized to exit from 192.42.116.24* network. Unless there is an economy behind the exit bandwidth, the network will not scale.

Or may be in future tunnels can go completely dark, but i am sure which direction the project will take.

More users in current state will only cripple the network.

**Comment by absolutep** [comment]
Then would it not be a good thing to allow more exit nodes to exist?

> Unless there is an economy behind the exit bandwidth

This thing crash landed so epicly many-a-times that developers swing between adding & removing it like a pendulum. At  The Moment, they have removed it.

> Or may be in future tunnels can go completely dark, but i am not sure which direction the project will take.

Have talked about this & opened tickets too, with the developers - no development in this regard till now.

> More users in current state will only cripple the network.

This network cannot survive without users. 

Furthermore, this network needs more users to develop & improve.

When even simple things such as formatting is not considered a priority then, trying to pursue & solve the unsolvable holds no real practical value.

**Comment by hbiyik** [comment]
Just trying to understand, why would anyone like to be an exit node?

In TOR's case, being volunteer for the "human rights" or "freedom of access" could somehow work because TOR's exit node bandwidth is quite less from a Tribler Exit node.

Considering even unencrypted vanilla torrent network is suffering from leeching problem, tunneling all the network to Exit nodes is simply not scalable to me. Currently only Delft University is being our "Robin Hood" or "Don Quijote" to provide such an exit possiblity. I am currently not sure if they are "Don Quijote" or "Robin Hood" :) time will tell.

**Comment by synctext** [comment]
>> Unless there is an economy behind the exit bandwidth

> This thing crash landed so epicly many-a-times that developers swing between adding & removing it like a pendulum. At The Moment, they have removed it.


Indeed, Tribler is the oldest experimental crypto economy. It works somewhat, but is not attack resilient. Certainly not a smooth user experience! We're getting quite close to a solution after 17 years and 8 months of work, called [MeritRank for fake identity fraud](https://arxiv.org/pdf/2207.09950) prevention. We have sufficient resources to grow to a million users, but after that this crypto economy needs to work. Exit node capacity is then indeed the constraining resource for privacy to manage efficiently.

To first actually to answer your question: we don't have global user statistics inside Tribler. We could easily do this, if we rely on our central statistics server for collecting global knowledge. No known method exists to do this in a decentralised manner reliably. For our "decentralised religion" reasons we don't want to depend on a hard-coded server in Tribler for any feature. I hope you understand that.

You touched upon on of the core scientific problems we're trying to solve within Tribler. Since Aug 2007 we have proposed the concept of Bandwidth-as-a-currency, see [BBC News exposure](http://news.bbc.co.uk/2/hi/technology/6971904.stm). Exit node capacity could be provided by a self-organising market. So, sort of VPN technology, but with credit coins and encryption. It turns out in those 17 years we had to re-create and re-invent a end-to-end financial infrastructure. 

< rant warning >
The strange thing is, our prototypes are more generic then I expected. Our Master students have even re-use some components to actually get an operational [prototype of a _digital Euro_](https://arxiv.org/pdf/2204.06831). 
[Our "upstoppable DAO" infrastructure](https://dl.acm.org/doi/pdf/10.1145/3565383.3566112) will one day be added to Tribler. Thus we would have Bittorrent, Tor-like priovacy, Bitcoin, and digital democracy into a single maximised decentralised platform.
Let me just say something completely ridiculous now. I think it is possible, in principle, to create a decentralised alternative for the global financial system. We can take away power from governments & companies and give this power back to citizens :fist_raised:. We are 15-25 years away from being able to replace the cardinal components of financial industry: [central marketplace monopolist CLS](https://www.swift.com/our-solutions/global-financial-messaging/messaging-solutions-for-fx-and-treasury/swift-and-cls) turnover is "$6 trillion per day in 18 currencies for 70 settlement members and over 30,000 third-party participants". 
My economist friend agrees with me, we can make the infrastructure both cheaper and more fault-tolerant. Thus you can dream realistically of a Napster-moment for finance. The music monopoly, encyclopedia monopoly, Hollywood monopoly, and OS monopoly once got disrupted. Lets end this whole "too big to fail" nonsense.

**Comment by qstokkink** [comment]
Perhaps this is two years too late but I opened a PR on Tribler's website repository that hooks in the `https://release.tribler.org/dashboard/` link under the `Statistics` again so users can see how many other users are in our ecosystem.
